tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -799,7 +798,6 @@ TEST(http_parser, chunked)
 	}
 
 	FOR_RESP("HTTP/1.1 200 OK\r\n"
-		 "Content-Length: 100\r\n"
```

Comment:
As we discussed yesterday, the cases should not be just removed. Rather the cases must be added to `mangled_messages()` with comment referencing to the RFC why the requests and responses are not valid. The test must ensure that the messages are blocked.